### ISSUE TITLE
Wrong scroll when setting the credit card number

### ISSUE THREAD

**Issue by GNZ** [dbachelder/CreditCardEntry#45]
Hi!

I'm having problems when I set the credit card number in the form:
This is if I don't focus the credit card number:
![Image1](http://i.imgur.com/7mlGxop.png)

And this is after focus the credit card number

![Image2](http://i.imgur.com/7B9GRCm.png)

If I enter a CVV it will validate it right. 

Very nice library btw.
 Regards

**Comment by dbachelder** [comment]
Hmm.. thanks for reporting.. but I'm not sure I understand.. can you reproduce this in the demo app?

**Comment by GNZ** [comment]
Yes, I reproduced it. The problem is that I set (programatically) a masked credit card number (Is what I received from the backend), it is consider as not valid and therefor it doesn't scroll correctly.
